Month3/Project1/Project1/ArrayVectors.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#define _USE_MATH_DEFINES
+#include <iostream>
+#include <iomanip>
+#include <cmath>
```

Comment:
Do you really need to include these here, in the header file? A recommended practice I learned in the Unreal courses was to avoid including things in the header that aren't necessary for the header itself. Include things in the .cpp file, where it is actually used. This is to avoid unnecessary imports when simply importing a class header file in another file entirely.

Reply:
Yessir! Avoid those includes in the header if you only need them in the cpp